src/Components/DirectionBox/DirectionBox.scss

```diff
@@ -39,7 +39,7 @@
     }
 
     .flag {
-      width: 70px;
+      transform: scale(1.3);
```

Comment:
The images are a little bit unsharp

Reply:
But this shouldn't be a problem with the new sprite as it has a higher resolution.